### ISSUE TITLE
Validate the size of the bundled browserify file.

### ISSUE THREAD

**Issue by arunoda** [meteorhacks/flow-router#128]
Check whether it's adding files we actually don't require.

**Comment by arunoda** [comment]
We had some issue with browserify since it add additional ~10KB. So, we removed browserify and manually added deps.

Now uncompressed file size is ~24KB.

**Comment by elidoran** [comment]
tl;dr -> upgrade cosmos:browserify to newest version and files are the same size.


You're still using cosmos:browserify version 0.1.3. The most current version is 0.3.0.

Most importantly, version 0.1.4 added specifying a debug value to the browserify call (see the CHANGES.md). Until then, it always passed debug:true and had a TODO comment saying I needed to determine when to pass true and when false. (by the way, arunoda is the one who suggested I check process.argv to determine production or dev. Thank you for the helpful idea.)

So, when using version 0.1.3 it still passes debug:true and the production output has the source map still in it.

If you upgrade the version passed 0.1.3 it will properly apply debug:false for both bundle/build (unless you specify --debug).

More details:

Browserify doesn't create a source map in a separate file like other tools do. It puts it in the browserified file.

The contents of `sourceMappingURL`, which is put in when `debug:true` is given to browserify, is a large amount of extra data. That doesn't exist when bundling/building for production, using the newer version of cosmos:browserify.

I tested the entire generated file `/packages/meteorhacks_flow-router.js` (dev version) for flow-router versions 1.12.0 and 1.13.0. The 1.12.0 (browserified) file is larger than including them yourself because of the source map. With the source map deleted it's smaller than including them yourself.

When I test the production generated file using cosmos:browserify 0.3.0 they are the same size for flow-router 1.12.0 and 1.13.0.

**Comment by arunoda** [comment]
Yep. You are right. Reverted :)
